packages/contracts/src/plugins/governance/admin/build-metadata.json

```diff
@@ -1,9 +1,21 @@
 {
   "ui": "",
```

Comment:
I think this field should be defined as an object, because I can see many fields within this section `ui: { entryPoint: "...", membersScript: "...", financeScript: "..."  }`

Unless we have a better way to be future proof

Reply:
Made it an object in f651606d87342a1cec6ff3bbc0dcad39bdfc6328